webpack/assets/javascripts/react_app/components/common/EmptyState/DefaultEmptyState.js

```diff
@@ -5,38 +5,37 @@ import SecondaryActionButtons from './EmptyStateSecondaryActionButtons';
 import { defaultEmptyStatePropTypes } from './EmptyStatePropTypes';
 import { translate as __ } from '../../../common/I18n';
 
-const documentationBlock = ({
-  url,
-  label = __('For more information please see'),
-  buttonLabel = __('Documentation'),
-}) =>
-  url && (
-    <React.Fragment>
-      {label}{' '}
-      <a href={url} target="_blank" rel="noopener noreferrer">
-        {buttonLabel}
-      </a>
-    </React.Fragment>
-  );
-
 const DefaultEmptyState = props => {
   const {
     icon,
     iconType,
     header,
     description,
-    documentation,
+    documentation: {
+      url,
+      label = __('For more information please see'),
+      buttonLabel = __('Documentation'),
+    } = {},
```

Comment:
Shouldn't we use `defaultProps` for those?

Reply:
I am not sure, since if someone will pass a new `documentation` prop, e.g: `{ url: 'some_url'}` 
but wants to use the default `label` and `buttonLabel` it will override all of the default props and they couldn't use the defaults we added.

another option would be to move this component into a different file and add the relevant props as default props there.

Reply:
Thanks. that make sense